.config/ansible-lint.yaml

```diff
@@ -1,6 +1,8 @@
 # vim:ts=2:sw=2:et:ai:sts=2
 ---
 profile: production
+exclude_paths:
+  - roles/external
```

Comment:
I'm not 100% sure about using roles/external, since that'll leak to downstream users (i.e. they'd need `role: tina_pm.common.external.resolv` instead of `role: tina_pm.common.resolv`). Thoughts/preferences?

Reply:
I am not sure either is preferred, since it is not really a tina.pm role..  I suppose there is no way to use the module with the collections name, or requiring the user to have cloned the repo somehow before? I don't really know, I have never worked with collections...

Reply:
Collections can only depend on other collections (by design, apparently). This means we'll have to vendor this roles _somewhere_.

> or requiring the user to have cloned the repo somehow before

This is what we're currently doing, and I'd like to change that. I'd like that our collection(s) can be installed as a single submodule, without having to manage symlinks, external repositories, or anything else. That's why I vendored it here.

Another option would be to create a separate collection (say, `tina_pm.third_party`), and vendor it there. Our downstreams would have to use two submodules instead of one, but I can live with that.

It involves a bit more maintenance, but it's also cleaner. What do you think?